modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/externalize/RelJsonReader.java

```diff
@@ -142,8 +144,17 @@ public RelTraitSet getTraitSet() {
         /** {@inheritDoc} */
         @Override
         public RelOptTable getTable(String table) {
-            List<String> list = getStringList(table);
-            return relOptSchema.getTableForMember(list);
+            throw new AssertionError();
```

Comment:
can u append additional messge into AssertionError ? do we really need _getTableById_ ? why can`t  refactor _getTable_ ?

Reply:
> can u append additional messge into AssertionError ?

Yes, of course. What exactly would you like to see in the message?

> do we really need getTableById ? why can`t refactor getTable ?

Someone could accidentally use it, so I would prefer to get a clear error, rather that UB (what if `IgniteUuid#fromString` starts to ignore leading and trailing braces or starts to return NULL in case it unable to parse the given input string?)

Reply:
```suggestion
            throw new AssertionError("Unexpected method was called.");
```
I understand that no additional info is appeared, but it`s just a little bit pretty, but it`s up to you.

Reply:
fixed